src/chains/eos_1_8/templates/chainActions/account_deleteAuth.ts

```diff
@@ -35,11 +35,11 @@ export const decomposeAction = (action: EosActionStruct): DecomposeReturn => {
   const { name, data, authorization } = action
 
   if (name === actionName && data?.account && data?.permission) {
-    const auth = getAuthorization(authorization)
+    const auth = getFirstAuthorizationIfOnlyOneExists(authorization)
```

Comment:
you forgot the comments about why we're using getFirstAuthorizationIfOnlyOneExists - should be in every function using it

Reply:
hmm seems like there has to be a better way. I think its being used in 15+ places, do we really want the same comment everywhere?

Reply:
well the point of the comment is to make it clear why it's being called. A person looking at the code would likely be looking at just one of the functions which is why we need it in each one.